src/dashboard/apps/core/mixins.py

```diff
@@ -0,0 +1,30 @@
+"""Dashboard core mixins."""
+
+from typing import Any
+
+from django.core.exceptions import PermissionDenied
+from django.http import Http404
+from django.shortcuts import get_object_or_404
+
+from apps.auth.models import DashboardUser
+from apps.core.models import Entity
+
+
+class EntityMixin:
```

Comment:
You need to state that it's a view mixin!

```suggestion
class EntityViewMixin:
```